example/upgrade_server.rb

```diff
@@ -39,12 +39,12 @@ def request_header_hash
 
 class UpgradeHandler
   VALID_UPGRADE_METHODS = %w(GET OPTIONS).freeze
-  UPGRADE_RESPONSE = <<-RESP
-HTTP/1.1 101 Switching Protocols
-Connection: Upgrade
-Upgrade: h2c
+  UPGRADE_RESPONSE = <<-RESP.strip_heredoc.freeze
```

Comment:
It might be feasible to use `<<~RESP` which achieves the same thing no? What version of Ruby does it go back to? (and does it really matter for this example client/server)

Reply:
I believe <<~ was introduced in 2.3. Technically, 2.1 and 2.2 are already EOL, so we _could_ require 2.3+ in the gem. That said, is it worth it?

Is Rubocop complaining about this one?

Reply:
Given that [2.2 is EOL](https://www.ruby-lang.org/en/news/2018/06/20/support-of-ruby-2-2-has-ended/). I'm OK with locking http-2 to 2.2+. If we go down this path, we should update the travis config and gemspec files as well.

Any objections? :)